### PR TITLE
Layer5 ng

### LINKED ISSUE #784: Create questionnaire
**Desired Situation**

Create a questionnaire (can use google forms connected to a spreadsheet) to invite requests for tutorials and for people interested in Hacktoberfest. The start of the form can be found [here](https://docs.google.com/forms/d/1uNEpOXWmMBk-d2fzARXgDEGOTCa3jDsbsEIfFUr4zJg/edit).

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by tripathyprateek**
**Description**
Added the Questionnaire Form
This PR fixes #784 

**Notes for Reviewers**
I havent enabled the link to which this should appear on the website. If needed, just tell me, I will add that.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "gatsby-plugin-manifest": "^2.2.30",
     "gatsby-plugin-mdx": "^1.2.27",
     "gatsby-plugin-react-helmet": "^3.1.16",
-    "gatsby-plugin-sharp": "^2.3.4",
+    "gatsby-plugin-sharp": "^2.6.36",
```
Comment (kushthedude) [inline package.json:33]: Please revert

File: src/sections/About/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ const About = () => {
                             {" "}
                             <h4>About us</h4>
                             <h2>
-                                <span>Starting with Layer5</span> is easier than anything.
+                                <span>Starting with Layer5</span> is easier than before.
```
Comment (kushthedude) [inline src/sections/About/index.js:37]: What if I am a new contributor? Please revert this

File: src/sections/Contact/index.js [hunk 3]
```diff
@@ -12,13 +12,13 @@ const Contact = () => {
             <SectionTitle className="section-title" UniWidth="55%">
                 <h4>Contact Us</h4>
                 <h2>
-                    <span>Drope your mail address </span> for next update.
+                    <span>Drop your email address </span> for updates from Layer5.
```
Comment (kushthedude) [inline src/sections/Contact/index.js:15]: Nice 🚀

File: src/sections/Questionnaire/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import "./styles.css";
```
Comment (kushthedude) [inline src/sections/Questionnaire/index.js:4]: Pass this as a component, why is this render itself?

**Review by kushthedude — CHANGES_REQUESTED** [review]
Few neats and changes to be addressed, nice going @tripathyprateek

**Comment by tripathyprateek** [comment]
> Few neats and changes to be addressed, nice going @tripathyprateek

Okay I will address all the changes as suggested.

**Comment by Tanuj22** [comment]
If I am not mistaken #784 was for the current layer site right? /cc @chandrashritii

**Comment by tripathyprateek** [comment]
> If I am not mistaken #784 was for the current layer5 site right? /cc @chandrashritii

Okay I will add it to Layer5 site as well.

File: src/sections/About/index.js (hunk 2 above)
Comment (leecalcote) [reply src/sections/About/index.js:37]: This area has yet to be updated from the original template, so we're essentially ok with anything here. We haven't done a first draft, yet.

File: src/sections/Questionnaire/index.js (hunk 4 above)
Comment (leecalcote) [reply src/sections/Questionnaire/index.js:4]: @tripathyprateek does this still need to be addressed?

File: package.json (hunk 1 above)
Comment (leecalcote) [reply package.json:33]: It it ok to move forward? Or does moving forward cause an unwanted breakage?

File: src/sections/Questionnaire/index.js (hunk 4 above)
Comment (tripathyprateek) [reply src/sections/Questionnaire/index.js:4]: Yes @leecalcote . I didnt have a very good grasp of Gatsby Js at the time of making this PR, and made few mistakes in the process. I will have them addressed and make a better one.

Comment (leecalcote) [reply src/sections/Questionnaire/index.js:4]: Oh, very good. 👍 No sweat. 😅

File: package.json (hunk 1 above)
Comment (tripathyprateek) [reply package.json:33]: It did show some warnings for me on the older version, and I ran `npm audit fix` and bumped up the dependencies.

Comment (leecalcote) [reply package.json:33]: Ok, unless @kushthedude comes back to advise otherwise, let's push forward with the higher version number.

**Comment by chandrashritii** [comment]
Good work @tripathyprateek Please solve the merge conflicts.

@Tanuj22 Yes. A link added to the current layer5.io site would be good too. We are going to add a card for Hecktoberfest on layer5/programs. I believe @jesuslerma is working on that. The form link can be added to that page.

**Comment by Tanuj22** [comment]
Closing as state

**Pull request closed without merging**